app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2 class="actions">Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
```

Comment:
- [ Optional ] I think it would be better if you style the `Login` form the same way it is styled in the project wireframe. The Login button should be the same `width` as the `email input` and `password input` fields.

## Your design
![Screenshot from 2023-06-22 23-35-07](https://github.com/atok624/Blog-app-rails/assets/16392046/1ef5c606-a0e7-46ff-9d6b-c9d48e190777)


## Project wireframe
![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)

Reply:
Thank you
I will implement that